packages/framework/src/Component/Router/NormalizeUrlTrailingSlashSubscriber.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Router;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\Routing\Exception\ResourceNotFoundException;
+
+class NormalizeUrlTrailingSlashSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Router\CurrentDomainRouter
+     */
+    protected $router;
```

Comment:
_Nitpick warning_

As this property is not for any router interface I would not name it so general.

```suggestion
    protected $currentDomainRouter;
```

It is longer but more describing in further context.

Reply:
Done